tools/install

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+ROOT=$(dirname $0)/..
+(cd $ROOT && npm install)
+(cd $ROOT && npm install)
+npm install -g bower
+(cd $ROOT/strategy-explorer && bower install)
```

Comment:
looks like we also need to bower install in `/devtools` (it has another readme)

Reply:
Done.